File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/library/items/media/files/ServiceFileUriQueryParamsProvider.java
package com.lasthopesoftware.bluewater.client.library.items.media.files;

public final class ServiceFileUriQueryParamsProvider implements IServiceFileUriQueryParamsProvider {

	@Override
	public String[] getServiceFileUriQueryParams(ServiceFile serviceFile) {
		return new String[]{
			"File/GetFile",
			"File=" + Integer.toString(serviceFile.getKey()),
			"Quality=medium",
			"Conversion=Android",
			"Playback=0"};
	}
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/items/media/files/ServiceFileUriQueryParamsProvider.kt
package com.lasthopesoftware.bluewater.client.browsing.items.media.files

object ServiceFileUriQueryParamsProvider : IServiceFileUriQueryParamsProvider {
	override fun getServiceFileUriQueryParams(serviceFile: ServiceFile): Array<String> = arrayOf(
		"File/GetFile",
		"File=" + serviceFile.key,
		"Quality=medium",
		"Conversion=Android",
		"Playback=0"
	)
}
